packages/pigeon/example/app/lib/src/messages.g.dart

```diff
@@ -29,6 +29,20 @@ List<Object?> wrapResponse(
   return <Object?>[error.code, error.message, error.details];
 }
 
+bool _deepEquals(Object? a, Object? b) {
+  if (a is List && b is List) {
+    return !a.indexed
+        .any(((int, dynamic) item) => !_deepEquals(item.$2, b[item.$1]));
```

Comment:
(1) nit: maybe more readable with `every`?
(2) what if `b` has more items than `a`?

Reply:
I spent some time looking for an 'all' method, but every is what I was hoping for, thanks for mentioning that.